novice-r-intro.Rmd

```diff
@@ -2,27 +2,108 @@
 
 # Introduction {#novice-r-intro}
 
+**Note: we will include this introduction in the novice Python material as well.**
+
 FIXME: general introduction.
 
 ## Who are these lessons for? {#novice-r-intro-personas}
 
 FIXME: personas
 
-### Summary
+## What will these lessons teach you? {#novice-r-intro-goals}
+
+During this course, you will learn how to:
 
-FIXME: elevator pitch
+-   Write programs in R that read data, clean it up, perform simple statistical analyses on it, and plot the results.
```

Comment:
Just a suggestion, maybe break this into two points and remove the mention of programs. 

- Use the R programming language to read, clean, and analyze data.
- Build meaningful visualizations to facilitate data exploration and to accurately and effectively communicate results.